applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -78,10 +70,7 @@ struct temp_cap_storage {
 
 static struct le_audio_headset headsets[CONFIG_BT_MAX_CONN];
 
-K_MSGQ_DEFINE(kwork_msgq, sizeof(struct worker_data),
-	      2 * (CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT +
-		   CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT),
-	      sizeof(uint32_t));
+K_MSGQ_DEFINE(cap_start_msgq, sizeof(uint8_t), CONFIG_BT_MAX_CONN, sizeof(uint8_t));
```

Comment:
Discuss: The last parameter is alignment, which here is 1. I looked at ~10 other uses of this macro, and they all use 4 for alignment.  This may be due to performance issues, as it is faster to address in multiples on 4 on ARM. Though, I have no other compelling reasons why we should have 4 here.

Reply:
The size is only 1 byte though

Reply:
I did a search and did find some other uses where alignment was 1. It is also stated in the documentation that it should be aligned by power of two, so 2^0 should be allowed ;)